### ISSUE TITLE
Improving single-image resolution with GAN

### ISSUE THREAD

**Issue by sam5658** [CognitiveLab-tech/World-of-AI#39]
## Project Request

---

| Generative AI  | Hi, I'm proposing this project which aims to improve the single-image resolution using the Super-Resolution Generative Adversarial Network (SRGAN) with the help of PyTorch.                     |
| ------ | --------------------------------- |
| About  | Generative Models such as SRGAN and ESRGAN have better loss functions than all the former GAN's resulting in generating images with higher resolution and better clarity, thus helping us improve the resolutions of low-quality images. |
| Github | Shubham Mishra         |
| Email  |  shubhammishra5658@gmail.com                               |
| Label  | Project Request                   |

---

**Define You**

- [x] GSSOC Participant
- [x] Contributor

# Project Name

## Description

[SRGAN is a generative adversarial network for single image super-resolution. It uses a perceptual loss function which consists of an adversarial loss and a content loss. The adversarial loss pushes the solution to the natural image manifold using a discriminator network that is trained to differentiate between the super-resolved images and original photo-realistic images. Generating Images with better higher resolutions ]

## Scope

[The project aims to improve the resolution of images with the help of a pytorch-based implementation.]

## Timeline

[starting date: assignment date
 duration: 5-10 days ]

## Video Links or Support Links

[[SRGAN](https://arxiv.org/pdf/1609.04802v5.pdf)]
